search_gov_crawler/search_gov_spiders/utility_files/crawl-sites.json

```diff
@@ -1,332 +1,324 @@
     },
     {
-        "allowed_domains": "uspto.gov",
-        "handle_javascript": false,
-        "starting_urls": "https://www.uspto.gov"
-    },
-    {
+        "name": "VA Access To Care",
+        "job_id": "va-access-to-care",
         "allowed_domains": "accesstocare.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.accesstocare.va.gov"
     },
     {
+        "name": "VA Benefits.va.gov",
+        "job_id": "va-benefits.va.gov",
         "allowed_domains": "benefits.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://benefits.va.gov/benefits/"
     },
     {
+        "name": "VA CFM Technical Information Library TIL",
+        "job_id": "va-cfm-technical-information-library-til",
         "allowed_domains": "cfm.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.cfm.va.gov/til/"
     },
     {
+        "name": "VA CIDER Research",
+        "job_id": "va-cider-research",
         "allowed_domains": "cider.research.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.cider.research.va.gov"
     },
     {
+        "name": "VA HERC Research - keep query strings",
+        "job_id": "va-herc-research-keep-query-strings",
         "allowed_domains": "herc.research.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.herc.research.va.gov/"
     },
     {
+        "name": "VA HSRD Research",
+        "job_id": "va-hsrd-research",
         "allowed_domains": "hsrd.research.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.hsrd.research.va.gov/"
     },
     {
+        "name": "VA My Health",
+        "job_id": "va-my-health",
         "allowed_domains": "myhealth.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.myhealth.va.gov"
     },
     {
+        "name": "VA Offica of Accountability and Whistleblower Protection",
+        "job_id": "va-offica-of-accountability-and-whistleblower-protection",
         "allowed_domains": "va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.va.gov/accountability/"
     },
     {
+        "name": "VA QUERI Research",
+        "job_id": "va-queri-research",
         "allowed_domains": "queri.research.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.queri.research.va.gov/"
     },
     {
+        "name": "VA Research & Development",
+        "job_id": "va-research-&-development",
         "allowed_domains": "research.va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.research.va.gov/"
     },
     {
+        "name": "VA Resources",
+        "job_id": "va-resources",
         "allowed_domains": "va.gov",
         "handle_javascript": false,
         "starting_urls": "https://www.va.gov/resources/"
     },
     {
+        "name": "DOD DANTES",
+        "job_id": "dod-dantes",
```

Comment:
nit: Do we really need this field? Can't we just convert the name to the `job_id` whenever we need, eg:
```python
name.lower().replace(" ", "-")
```

Unless we will need to use it as a unique identifier or as a reference later on